### ISSUE TITLE
RandomTeleportation module try to store its option under wrong config level

### ISSUE THREAD

**Issue by mhtvsSFrpHdE** [Zrips/Residence#514]
Residence version: `4.9.1.1`
Minecraft server version:
```
Spigot, CraftBukkit version git-Spigot-a99063f-be6aaf0 (MC: 1.15.2) (Implementing API version 1.15.2-R0.1-SNAPSHOT
```

![image](https://user-images.githubusercontent.com/10773245/87338448-ca0f4580-c577-11ea-9537-9ad7e6463649.png)
I put my server config into a Git repository to track changes  
Then discovered this

Initialize plugin config once, create a commit  
Shutdown then reboot the server, strange change appears in Git diff

**Comment by matbcvo** [comment]
It's fixed in 4.9.1.3, check #499

**Comment by Zrips** [comment]
0144cfb053feb1f0f04907d24f283e0e90a904d3
